src/Munee/Asset/Type/Css.php

```diff
@@ -204,4 +205,42 @@ protected function fixRelativeImagePaths($content, $originalFile)
 
         return $content;
     }
+
+    private static function parseImports($content,$origFile)
+    {
+        $dir   = dirname($origFile);
+        // matches any type of import rule
+        preg_match_all('~@import\s*(?:url)?(?:\(?\'?\"?)?([^\'\"\)\(]*)(?:\'?\"?)?\)?\s?([^;]*);~im',$content,$imports,PREG_SET_ORDER);
+        if(count($imports))
+        {
+            foreach($imports as $i => $item)
+            {
+                $file  = $dir.'/'.$item[1];
+                $media = $item[2];
+                    if (is_file($file))
+                    {
+                        $string = file_get_contents($file);
+                        $newDir = dirname($file);
+                        // replace imports in current file
+                        $string = self::parseImports($string,$file);
+                        // replace urls
+                        if ($newDir !== $dir) {
+                            $tmp = $dir.'/';
+                            if (substr($newDir, 0, strlen($tmp)) === $tmp) {
+                                $string = preg_replace('#\burl\(["\']?(?=[.\w])(?!\w+:)#', '$0' . substr($newDir, strlen($tmp)) . '/', $string);
+                            }
+                        }
+                    if($media && $media != '')
```

Comment:
empty() is not the same, as a content of 0 would be empty, too. ;) I tend to be very strict.